acme/acme/messages.py

```diff
@@ -92,10 +93,10 @@ def code(self):
             return code
 
     def __str__(self):
```

Comment:
This method now returns a `unicode` object in Python 2. It seems to work fine, but Python's docs say that the return value of `__str__` must be a string object ([source](https://docs.python.org/2/reference/datamodel.html#object.__str__)) and I think we should follow that just in case not doing so will cause issues down the road.

The easiest fix that I'm aware of is to use [@six.python_2_unicode_compatible](https://pythonhosted.org/six/#six.python_2_unicode_compatible). This will have no effect in Python 3 but will call `.encode('utf-8')` on the return value in Python 2.